### DISCUSSION TITLE
Release 2.2.3

### DISCUSSION THREAD

**Discussion by androidacy-user** [Androidacy/MagiskModuleManager#62] [General]
**Full Changelog**:  https://www.androidacy.com/amm-2-2-3-the-ultimate-magisk-module-updater/

Downloads are linked in the post.

<hr /><em>This discussion was created from the release <a href='https://github.com/Androidacy/MagiskModuleManager/releases/tag/v2.2.3'>Release 2.2.3</a>.</em>

**Comment by Wojtaz0w** [comment]
Switches in security, privacy and debug settings are MD2! Please correct them. Also transitions in srttings arent animated 🫤
